src/test/scala/uk/gov/hmrc/perftests/dds/Simulation.scala

```diff
@@ -0,0 +1,102 @@
+/*
+ * Copyright 2022 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package uk.gov.hmrc.perftests.dds
+
+import uk.gov.hmrc.performance.simulation.PerformanceTestRunner
+import uk.gov.hmrc.perftests.dds.requests.AddressLookupRequests._
+import uk.gov.hmrc.perftests.dds.requests.AuthLoginStubRequests._
+import uk.gov.hmrc.perftests.dds.requests.DigitalDisclosureRequests._
+
+class Simulation extends PerformanceTestRunner {
+
+  setup(
```

Comment:
Simulations should be split into parts for readability and to reduce code duplication. e.g. `auth-login`, and `address-lookup` can be used across as many journeys as you like without defining the same requests repeatedly.